pages/pipelines/tags.md

```diff
@@ -0,0 +1,13 @@
+# Pipeline tags
+
+Pipeline tags allow you to tag and search for your pipelines using the search bar. Tags are beneficial when you have many pipelines and would like to group and filter through them quickly.
+
+<%= image "pipeline-tag-search.png", alt: "Imagine showing pipeline tag search" %>
+
+## Using tags
+
+Tags are unique per pipeline. You can only add the same tag once. Tags are limited to five per pipeline, and are no longer than 64 chars. Tag names support emoji and text.
+
+<%= image "pipeline-tag-settings.png", alt: "Imagine showing pipeline tag settings" %>
```

Comment:
I'm on a bit of a mission to avoid adding unnecessary screenshots to make the docs easier to maintain. I think this one could be removed without losing much value. What do you think?

Reply:
Absolutely - if it doesn't add value, let's remove!